test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.